workspace/viewer/PdfJsViewer.ts

```diff
@@ -387,10 +393,9 @@ export class PdfJsViewer {
       const onDocumentSuccess = (pdf) => {
         this.currentFile = pdf;
         this.currentFileUrl = task.url;
-        if (this.currentPage > pdf.numPages) {
```

Comment:
Why did you remove this? `currentPage` is supplied externally and it *really* can be greater than the number of pages. And no matter if we have smooth scrolling or not, we'll want to show user that very `currentPage`.

Reply:
This part of code loses relevance if we want to load and show more than one document's page, because it will refer to last loaded page. So I moved this logic to another place.

Reply:
Can you explain this? 

> because it will refer to last loaded page

`currentPage` is not the last loaded, it is the page user had been looking at until we reloaded the document. If user had been looking at page 10 and in the new PDF we have only 9 pages, we have to set `currentPage` properly.

Reply:
From this point, you're right. I wanted to update currentPage from scroll event, so loading new PDF would reset it.